src/utils/labelUtils.ts

```diff
@@ -131,6 +131,22 @@ const isQualifiedName: IValidationFunction = (key) => {
   };
 };
 
+export const labelKeyIsRestricted = (key: string) => {
+  return Constants.RESTRICTED_CLUSTER_LABEL_KEYS.some(
+    (restrictedKeyItem) => key.toLowerCase() === restrictedKeyItem
+  );
+};
```

Comment:
Since these utility functions return booleans, what do you think about using the naming scheme `is...` or `has...`?

Reply:
Hmm that’s a harder one.. What about`hasRestrictedLabelKeySubstring` or `hasRestrictedSubstringForLabelKey`? If that’s too long I think `hasRestrictedSubstring` would work too in the context